README.md

```diff
@@ -154,18 +158,26 @@ For further details, refer to the [Wagtail API v2 Configuration Guide](https://d
 </html>
 ```
 
+
 * Install django-cors-headers: `pip install django-cors-headers`
-* Add to your settings file 
+* Add CORS config to your settings file to allow the front-end to access the api
+
 ```python
 CORS_ORIGIN_ALLOW_ALL = True
 CORS_URLS_REGEX = r'^/api/v2/'
- ``` 
+ ```
 
 * Start up your site as normal: `./manage.py runserver 0:8000`
-* Set up `client/index.html` to be served at `http://localhost:8020/` - 
-  this can be done by running `python3 -m http.server 8020` from inside the client directory
-* Edit (or created) and preview a page that uses `HeadlessPreviewMixin`
-* The preview page should now show you the API response for the preview
+
+Now we can access the preview from our barebones demo front-end:
+
+* Serve `client/index.html` at `http://localhost:8020/`
+   - this can be done by running `python3 -m http.server 8020` from inside the client directory
+* From the wagtail admin interface, edit (or create) and preview a page that uses `HeadlessPreviewMixin`
+
+The resultant preview page should now show you the API response for the preview! 🎉
+
+This is where a real front-end would take over and display the preview as it would be seen on the live site.
```

Comment:
wagtail ➜ Wagtail

> Now we can access the preview from our barebones demo front-end:

This is not true until the "client" is up and running

"The resultant preview page" ➜ "The preview page"